### ISSUE TITLE
Better test coverage for CI

### ISSUE THREAD

**Issue by jgliss** [jgliss/pyplis#29]
Running the example scripts (which include high level tests ensuring the basic functionality) is too heavy and requires downloading the official test dataset (> 2GB) which is used in the paper. This is not optimal for the CI suite, since this would always trigger the download (which may take long) and also, running the scripts itself is not the fastest. 
I suggest to include a lightweight version of these tests in the pytest environmen (as official test modules) and based on the data in `pyplis/data/testdata_minimal` (cf. discussion in #27).

**Comment by johannjacobsohn** [comment]
Agreed, downloading 2G might be too much for travis... I didn't consider that. I just fear that not running them automatically might lead to not running them at all, which will lead to breakage (both in scripts and in pyplis, as the scripts cover a lot more ground).
To increase test coverage it might be a good idea to add test coverage testing in first, so that we can track improvement of test coverage (as well as test distribution)
